config/packages/flysystem.yaml

```diff
@@ -0,0 +1,6 @@
+flysystem:
+    storages:
+        default.storage:
+            adapter: 'local'
+            options:
+                directory: '%kernel.project_dir%/var/uploads'
```

Comment:
The tests are failing now because this is a userland configuration. What should I do with that? Move it to a "prependExtension" of the media bundle or add this config to all bundles that have a dependency on the media bundle?